src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/ApplicationLogic.cs

```diff
@@ -26,5 +26,12 @@ public class ApplicationLogic
         /// </summary>
         [JsonProperty(PropertyName = "schemaRef")]
         public string SchemaRef { get; set; }
+
+        /// <summary>
+        /// Specifies wether anonymous access is allowed in stateless mode or not for this particular data type.
```

Comment:
Thank you I learned something today. A wether is a a castrated ram. 
Try whether. You can also consider removing "or not", it's implied.